jenkins/metadata-cache/metadata-cache-control.sh

```diff
@@ -94,6 +105,10 @@ do_local() {
       ;;
     start)
       echo -n "Starting metadata-cache session: "
+      if [ -x "$(which metadata-cache.py)" ]; then
```

Comment:
Two square braces as per [the style guide](https://google.github.io/styleguide/shell.xml#Test,_%5B_and_%5B%5B).